<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCL_setting_moduleClass</class>
 <widget class="QWidget" name="PCL_setting_moduleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCL_setting_module</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_218">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Color Lookuptable:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="lookuptable_color_comboBox">
         <item>
          <property name="text">
           <string>Jet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Parula</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hsv</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Viridis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cool</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Spring</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Summer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Autumn</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Winter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gray</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bone</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Copper</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lines</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1">
       <item>
        <widget class="QCheckBox" name="auto_x_checkbox">
         <property name="text">
          <string>auto-X </string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="min_x_doubleSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-200.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>-200.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="__label_3">
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="max_x_doubleSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-200.000000000000000</double>
         </property>
         <property name="maximum">
          <double>800.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>800.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1">
       <item>
        <widget class="QCheckBox" name="auto_y_checkbox">
         <property name="text">
          <string>auto-Y </string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="min_y_doubleSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-200.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>-200.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="__label_4">
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="max_y_doubleSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-200.000000000000000</double>
         </property>
         <property name="maximum">
          <double>800.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>800.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1">
       <item>
        <widget class="QCheckBox" name="auto_z_checkbox">
         <property name="text">
          <string>auto-Z </string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="min_z_doubleSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-300.000000000000000</double>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
         <property name="value">
          <double>-50.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="__label">
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="max_z_doubleSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-300.000000000000000</double>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>50.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QCheckBox" name="grid_on_checkbox">
         <property name="text">
          <string>Grid on</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="colorbar_on_checkbox">
         <property name="text">
          <string>Colorbar</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
       <item>
        <widget class="QLabel" name="lut_label_xyz">
         <property name="text">
          <string>Color - XYZ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="color_xyz_comboBox">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>View Conut</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1,0,1">
       <item>
        <widget class="QCheckBox" name="auto_lut_xyz_checkbox">
         <property name="text">
          <string>auto</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="min_xyz_lut_doubleSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="__label_5">
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="max_xyz_lut_doubleSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
       <item>
        <widget class="QLabel" name="point_size_label">
         <property name="text">
          <string>Point size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="point_size_spin">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1">
       <item>
        <widget class="QLabel" name="text_scale_label">
         <property name="text">
          <string>Text scale:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="text_scale_spin">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
       <item>
        <widget class="QLabel" name="ori_scale_label">
         <property name="text">
          <string>Ori size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="ori_scale_spin">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>0.000010000000000</double>
         </property>
         <property name="maximum">
          <double>100000000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="code_checkBox">
         <property name="text">
          <string>Code</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="path_checkBox">
         <property name="text">
          <string>Show Path</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ori_checkBox">
         <property name="text">
          <string>Show Ori</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
